code/__DEFINES/admin.dm

```diff
@@ -94,7 +94,7 @@
 #define AHELP_RESOLVED 3
 
 /// Amount of time (in deciseconds) after the rounds starts, that the player disconnect report is issued.
-#define ROUNDSTART_LOGOUT_REPORT_TIME 6000
+#define ROUNDSTART_LOGOUT_REPORT_TIME 12600
```

Comment:
This dispatches the roundstart logout report to admins 21 minutes after the round starts as well, that's a bit too long. More on that later.

I recall the backend of the roundstart logging system (which is probably why you doubled it like this) being a bit weird, I don't think this is the smartest of ideas to have defines underutilized like this. Might be more apt to do something like:

```dm
/// Amount of time that we dispatch the round-end report to admins.
#define ROUNDSTART_LOGOUT_REPORT_TIME (15 MINUTES)
/// The duration of time it takes for someone to be considered logged out for the round-end logout report.
#define ROUNDSTART_LOGOUT_REPORT_THRESHOLD (ROUNDSTART_LOGOUT_REPORT_TIME / 2)
```

Much easier to use the minutes define here rather than do it in raw deciseconds. The latter define would be used here, for example:

![image](https://user-images.githubusercontent.com/34697715/221448237-b5654624-7fd1-46ab-8579-40548cdd8b05.png)

Don't have to follow these define names or values, but would be nice if you could comb through the code and substitute these values in instead so we have some uniform consistency on tying being AFK and being put in that report to the define system rather than perform arbitrary-ish calculations in the code. It'll be much clear why we're doing what we're doing in the code when someone comes through and passes stuff through AFK logging.

Sorry if that's a bit confusing, but I think we can freshen up the whole code in the area where this define is already used

I don't think the round-end logout report should be shown more than 15 minutes after the round starts... the shuttle can already be called by the time we get the "round start log out report".

Reply:
Sure I'll take a look.

Reply:
I moved the define and reduced the timer to 10 minutes, with a 70% threshold for AFK.